app/models/job_invocation_composer.rb

```diff
@@ -98,6 +121,12 @@ def template_with_default
       return template
     end
 
+    private
+
+    def format_datetime(datetime)
+      return datetime if datetime.blank?
+      DateTime.parse(datetime).strftime('%Y-%m-%d %H:%M')
```

Comment:
Yes probably but I think the API controller should bubble the message up to the user no? Will test it and see